### DISCUSSION TITLE
Finding the right balance between blocking with hosts file and AdNauseam

### DISCUSSION THREAD

**Discussion by ghost** [dhowe/AdNauseam#1948] [General]
I'm working on a hardened set-up of Debian which includes a massive hosts file blocking over 1,000,0000 domains (updated by a script I found to download and collate hosts lists) with the purpose of saving a huge amount of bandwidth and performance and minimizing the amount of filters enabled in uBlock Origin (Which I hope reduces my browser signature).

Here are SOME of the filter lists whose contents I added to my hosts file.

StevenBlack's Unified Hosts file
https://raw.githubusercontent.com/StevenBlack/hosts/master/alternates/fakenews-gambling-porn-social/hosts

Dandelion's Anti-Malware List
https://raw.githubusercontent.com/DandelionSprout/adfilt/master/Alternate%20versions%20Anti-Malware%20List/AntiMalwareHosts.txt

Combined Privacy Block Lists
https://raw.githubusercontent.com/bongochong/CombinedPrivacyBlockLists/master/newhosts-final.hosts

However, I recently came upon AdNauseam and was drawn by the idea of obfuscating my browsing traffic by automatically clicking ads... Ads that will have already been prevented from being downloaded in the first place by my hosts file so that AdNauseam can't "click" them.

So now I'm left wondering what the perfect balance between blocking with the hosts file and blocking with AdNauseam would be.
I've quickly put together a chart of all the filter lists I'm considering but it's still in the works.
[Blocklist Planning.pdf](https://github.com/dhowe/AdNauseam/files/7345337/Blocklist.Planning.pdf)

What's everyone's thoughts here?
Thanks.

**Comment by Enrra44** [comment]
Hi, 
I am not at all an expert of programming, especially in online ad blocking but here is a thought : 
- Could you let the ad being downloaded the first time you see it on a website, let Adnauseam click on it, then "unload" it ? 
Or at least prevent them for being downloaded the next time ? 
I don't know if it is any possible or if it is even usefull but could work
Have a nice day
